rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1187,12 +1188,12 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 		});
 		connect(ui->gs_width, &QSpinBox::editingFinished, [=]()
 		{
-			ui->gs_width->setValue(std::min(ui->gs_width->value(), QApplication::desktop()->screenGeometry().width()));
+			ui->gs_width->setValue(std::min(ui->gs_width->value(), QGuiApplication::primaryScreen()->size().width()));
```

Comment:
why do you use size here and geometry above?

Reply:
I want to keep the line length reasonable, and geometry is twice as long. The difference between size and geometry is that geometry has a starting point, they have identical width and height.

Reply:
Too tired to think about all of this.
I'll just trust that everything just works